README.md

```diff
@@ -2,14 +2,32 @@
 ## Abstract
 
 VMware Cloud Director Object Storage Extension (for short, OSE) is a standalone middleware service installed in private data center or public cloud to provide object storage capabilities to the users of VMware Cloud Director.
-OSE supports three object storage platforms out of box: Cloudian, Dell EMC ECS and Amazon S3.
 
 OSIS (Object Storage Interoperability Service) is proposed to extend OSE to support other object storage platforms by defining unified administrative interfaces for storage platforms.
 
 For the platforms integrated with OSE via OSIS, the data channel is between OSE and the platform, but the control channel is between OSE and OSIS implementation (REST services implementing OSIS).
 
 This project is for OSIS which integrates [Scality RING](https://www.scality.com/products/ring/) with vCloud Director OSE. 
 
+## Running the Application
+
+### Command to run Docker Image
+
+Run `docker run --env-file application.properties -it -p8443:8443 osis-scality-0.1.0-snapshot`
+
+Sample `application.properties` file can be found in GIT at `/src/main/resources/application.properties`
+
+#### To view Tomcat Access logs in the docker container
+* Run `docker exec -it <Container_ID> /bin/bash`
+* Access log files can be found under `/tomcat/logs` 
+
+### Steps to run application as a standalone jar (Developers Only) 
+1. Update the 
+2. `./gradlew clean build`
+3. `./gradlew bootJar`
+4. `java -jar -Dserver.tomcat.basedir=tomcat -Dserver.tomcat.accesslog.directory=logs -Dserver.tomcat.accesslog.enabled=true build/libs/osis-scality-0.1.0-SNAPSHOT.jar`
```

Comment:
Can we also make the version generic here?
As I tried these commands it produced `osis-scality-0.0.1-SNAPSHOT.jar`

Reply:
Updated to generic version